dogstatsd/entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#set -e
+
+if [[ $DD_API_KEY ]]; then
```

Comment:
if we're going to run the forwarder it makes sense to use supervisor, but I think we should keep this entrypoint small. Most options in the agent one are irrelevant here and could even be misleading (for example if users see `if [[ $TAGS ]]; then` here they might think we can apply tags to dogstatsd metrics this way.